pystiche/papers/ulyanov_et_al_2016/data.py

```diff
@@ -0,0 +1,269 @@
+    content_base_ulyanov = urljoin(base_ulyanov, "readme_pics/")
+    content_images = {
+        "karya": DownloadableImage(
+            urljoin(content_base_ulyanov, "karya.jpg"),
+            md5="232b2f03a5d20c453a41a0e6320f27be",
+        ),
+        "tiger": DownloadableImage(
+            urljoin(content_base_ulyanov, "tiger.jpg"),
+            md5="e82bf374da425fb2c2e2a35a5a751989",
+        ),
+        "neckarfront": DownloadableImage(
+            "https://upload.wikimedia.org/wikipedia/commons/0/00/Tuebingen_Neckarfront.jpg",
+            title="Tübingen Neckarfront",
+            author="Andreas Praefcke",
+            license=CreativeCommonsLicense(("by",), version="3.0"),
+            md5="dc9ad203263f34352e18bc29b03e1066",
+            file="tuebingen_neckarfront__andreas_praefcke.jpg",
+        ),
+        "CheHigh": DownloadableImage(
+            "https://upload.wikimedia.org/wikipedia/commons/5/58/CheHigh.jpg",
+            title="CheHigh",
+            author="Alberto Korda",
+            date="1960",
+            license=PublicDomainLicense(1960),
+            md5="cffc0768090c5a705cbb30fdc24c3e64",
+            file="CheHigh.jpg",
+        ),
+        "The_Tower_of_Babel": DownloadableImage(
+            "https://upload.wikimedia.org/wikipedia/commons/f/fc/Pieter_Bruegel_the_Elder_-_The_Tower_of_Babel_%28Vienna%29_-_Google_Art_Project_-_edited.jpg",
+            title="The Tower of Babel",
+            author="Pieter Bruegel",
+            date="1563",
+            license=PublicDomainLicense(1563),
+            md5="1e113716c8aad6c2ca826ae0b83ffc76",
+            file="the_tower_of_babel.jpg",
+        ),
+        "bird": DownloadableImage(
+            urljoin(base_ulyanov_suppl, "bird.jpg"),
+            md5="74dde9fad4749e7ff3cd4eca6cb43d0d",
+        ),
+        "kitty": DownloadableImage(
+            urljoin(readme_ulyanov, "kitty.jpg"),
+            md5="98262bd8f5ae25f8329158d2c2c66ad0",
+        ),
+    }
+
+    texture_base_ulyanov = urljoin(base_ulyanov, "textures/")
+    base_ulyanov_suppl_texture = "https://raw.githubusercontent.com/DmitryUlyanov/texture_nets/texture_nets_v1/supplementary//texture_models/"
+
+    texture_images = {
+        "cezanne": DownloadableImage(
+            urljoin(texture_base_ulyanov, "cezanne.jpg"),
+            md5="fab6d360c361c38c331b3ee5ef0078f5",
+        ),
+        "bricks": DownloadableImage(
+            urljoin(base_ulyanov_suppl_texture, "bricks.png"),
+            md5="1e13818e1fbefbd22f110a1c2f781d40",
+        ),
+        "pebble": DownloadableImage(
+            urljoin(base_ulyanov_suppl_texture, "pebble.png"),
+            md5="5b5e5aa6c579e42e268058a94d683a6c",
+        ),
+        "pixelcity_windows2": DownloadableImage(
+            urljoin(base_ulyanov_suppl_texture, "pixelcity_windows2.jpg"),
+            md5="53026a8411e7c26e959e36d3223f3b8f",
+        ),
+        "red-peppers256.o": DownloadableImage(
```

Comment:
The identifier does not have to be same as the file name but rather should be intuitive. Thus, please remove the unneeded information.